III.tdd-katas/calculator-tdd/add.test.js

```diff
@@ -0,0 +1,40 @@
+const add = require("./add");
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
+        expect(add("")).toBe(0);
+    });
+    test('should return 1 for "1" string passed to add', () => {
+        expect(add("1")).toBe(1);
+    });
+    test('should return 3 for "1,2" string passed to add', () => {
+        expect(add("1,2")).toBe(3);
+    });
+});
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
+        expect(add("1,2,4fi5faswwe6,lgsf8")).toBe(26);
+    });
+});
+describe("Step 3: handle new lines between numbers", () => {
+    test("should return 26 for the string passed to add", () => {
+        expect(add("1,2,4fi5faswwe6,lgsf8")).toBe(26);
```

Comment:
This step describes handing newlines, but your test input doesn't contain any.

Reply:
I am extremely sorry for not replying to your comments. Thanks very much for such a long comments and I will do the changes as you pointed out. It was a fantastic class last week and learnt a lot of new things, especially the approach of TDD you showed step by step with the real example!